script.module.enum34/lib/enum/enum/__init__.py

```diff
@@ -0,0 +1,837 @@
+"""Python Enumerations"""
```

Comment:
this looks identical to the other `__init__.py`. What is the difference?

Reply:
This code is by the original author whose package has been adopted into Python 3.4+

Reply:
This code was provided by the original author of the enum package.
[enum download](https://pypi.org/project/enum34/)
This package is being distributed as part of Python 3.4+